# Generated for NaoQi method ALMotion.killAll
# Kills all tasks.

---


